### PR TITLE
Feature/set namespace

### PR DESCRIPTION
**Pull request by alaminopu**
Changes related to issue #5

### CODE REVIEW

**Comment by alaminopu** [comment]
@dnivra26 need review?

File: src/main.rs [hunk 1]
```diff
@@ -31,9 +33,22 @@ fn main() {
             .long("namespace"))
         .get_matches();
     
-    let namespace = matches.value_of("namespace").unwrap();
+    let namespace_from_config = config::get();
+    let mut namespace = matches.value_of("namespace").unwrap();
+    if !namespace_from_config.is_empty() && namespace == "default" {
```
Comment (dnivra26) [inline src/main.rs]: what if the user specifies "default" as the namespace using "-n". which one should get more precedence? config file or -n argument?

Comment (alaminopu) [reply src/main.rs]: @dnivra26 I think -n argument should always have higher precedence. Let me fix that.

Comment (alaminopu) [reply src/main.rs]: @dnivra26 Fixed!

**Pull request merged**